build_tools/cmake/iree_copts.cmake

```diff
@@ -420,7 +420,7 @@ set(LLVM_ENABLE_IDE ON CACHE BOOL "" FORCE)
 # TODO(ataei): Use optional build time targets selection for LLVMAOT.
 set(LLVM_TARGETS_TO_BUILD "WebAssembly;X86;ARM;AArch64;RISCV" CACHE STRING "" FORCE)
 
-set(LLVM_ENABLE_PROJECTS "mlir" CACHE STRING "" FORCE)
+set(LLVM_ENABLE_PROJECTS "mlir;compiler-rt" CACHE STRING "" FORCE)
```

Comment:
what's the compile time cost of this? is it required? (if the user is building their application with asan they aren't using this copy of compiler-rt, and conflicting will be bad)

Reply:
It was trial when I was trying to test it with ASAN libraries.. And forgot to remove it. Sorry and thanks for letting me know.